KamerConnect.DataAccess.Postgres/Repositories/ChatRepository.cs

```diff
@@ -96,4 +96,102 @@ private ChatMessage ReadToChatMessage(DbDataReader reader)
             reader.GetDateTime(4)
         );
     }
+        
+    public void CreateChat(Guid chatId)
```

Comment:
CreateChat zou ik private maken. Dan 1 public variant die de chat zelf aanmaakt en de persons en dat dan in 1 transaction. Zie mijn geolocation PR of de house repository